### PR TITLE
modify-2.0-build

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
### What is changed, added or deleted? (Required)

- Modified some lints to fit mkdocs build.
- Rename some fiels.

### CODE REVIEW

File: docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md [hunk 1]
```diff
@@ -138,27 +138,26 @@ No such attention is needed when matching on ID because the index for vertex IDs
...
-    ```
+```nGQL
+nebula> MATCH (v) WHERE id(v) == 'player101' RETURN v;
++---------------------------------------------------+
+| v                                                 |
++---------------------------------------------------+
+| (player101) player.name:Tony Parker,player.age:36 |
++---------------------------------------------------+
+Got 1 rows (time spent 1710/2406 us)
+```
```
Comment (oldLady344) [inline docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md:149]: need some indent?

File: docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md [hunk 2]
```diff
@@ -138,27 +138,26 @@ No such attention is needed when matching on ID because the index for vertex IDs
...
+nebula> MATCH (v) WHERE id(v) IN ['player100', 'player101', 'no_one'] RETURN v;
++---------------------------------------------------+
+| v                                                 |
++---------------------------------------------------+
+| (player101) player.name:Tony Parker,player.age:36 |
++---------------------------------------------------+
+| (player100) player.name:Tim,player.age:42         |
++---------------------------------------------------+
+Got 2 rows (time spent 1307/2079 us)
+```
```
Comment (oldLady344) [inline docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md:163]: need some indents?

**Review by oldLady344 — APPROVED** [review]

File: docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md (hunk 2 above)
Comment (Amber1990Zhang) [reply docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md:163]: no indents here or something is wrong with the build

**Review by randomJoe211 — APPROVED** [review]

File: docs-2.0/3.ngql-guide/6.functions-and-expressions/5.case-expressions.md [hunk 3]
```diff
@@ -35,12 +35,12 @@ END
 
     ```nGQL
     nebula> YIELD \
-         -> CASE 2+3 \
-         -> WHEN 4 THEN 0 \
-         -> WHEN 5 THEN 1 \
-         -> ELSE -1 \
-         -> END \
-         -> AS result;
+          CASE 2+3 \
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/6.functions-and-expressions/5.case-expressions.md:38]: good catch

File: docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md (hunk 1 above)
Comment (whitewum) [reply docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md:149]: indent will cause display error

**Pull request merged**